product-scenarios/4-application-development-using-apis/4.1-application-developers-should-be-able-to-use-apis-in-their-applications/4.1.3-delete-registered-application/src/test/java/org/wso2/am/scenario/tests/delete/registered/application/DeleteRegisteredApplicationNegativeTestCase.java

```diff
@@ -0,0 +1,71 @@
+package org.wso2.am.scenario.tests.delete.registered.application;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.am.integration.test.utils.base.APIMIntegrationConstants;
+import org.wso2.am.scenario.test.common.APIStoreRestClient;
+import org.wso2.am.scenario.test.common.ScenarioTestBase;
+import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+
+public class DeleteRegisteredApplicationNegativeTestCase extends ScenarioTestBase {
+    private APIStoreRestClient apiStore;
+    private static final Log log = LogFactory.getLog(DeleteRegisteredApplicationNegativeTestCase.class);
+    private static final String ADMIN_LOGIN_USERNAME = "admin";
+    private static final String ADMIN_LOGIN_PW = "admin";
+    private static final String APPLICATION_NAME = "DeleteUnownedApplication";
+    private static final String SUBSCRIBER_USERNAME = "deleteAppSubscriber1";
+    private static final String SUBSCRIBER_PW = "deleteAppSubscriber1";
+    private static final String SUBSCRIBER2_USERNAME = "deleteAppSubscriber2";
```

Comment:
Why do we need these constants as static?

Reply:
Removed static constants